lib/utils.js

```diff
@@ -36,7 +36,9 @@ export function grammarToLanguage(grammar: ?atom$Grammar) {
     l => l.toLowerCase() === grammarLanguage
   );
 
-  return kernelLanguage ? kernelLanguage.toLowerCase() : grammarLanguage;
```

Comment:
[`_.findKey`](https://lodash.com/docs/4.17.4#findKey) already returns the mapped language.